.github/workflows/release_updates.yml

```diff
@@ -0,0 +1,24 @@
+# Automatically updates "vX" branches based on GitHub releases. To cut a new
+# release, use the GitHub UI where you enter a tag name and release name of
+# "vX.Y.Z". See https://github.com/jupyterhub/action-k3s-helm/releases.
+#
+#
+name: Release updates
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
+    steps:
+      # Action reference: https://github.com/Actions-R-Us/actions-tagger
+      - uses: Actions-R-Us/actions-tagger@latest
+        env:
+          GITHUB_TOKEN: "${{ github.token }}"
+        with:
+          # By using branches as identifiers it is still possible to backtrack
+          # some patch, but not if we use tags.
+          prefer_branch_releases: true
+          publish_latest_tag: true
```

Comment:
What are your thoughts on not having `latest`? I can't think of a good reason for it since it'll mean unexpected breaking changes for users.

Reply:
I think of it as an option in between `@main` and `@vX` that could be valid for someone that wants to stay on the bleeding edge. I don't think it matters though, I will use `@vX` + dependabot to bump X to X+1 instead.

Here is a button for your convenience :D

```suggestion
```